.changeset/large-needles-serve.md

```diff
@@ -0,0 +1,31 @@
+---
+'sku': minor
+---
+
+Add support for Storybook's `preview.js` file
+
+sku now supports global customization of story rendering via a `.storybook/preview.(js|ts|tsx)` file.
+
+**EXAMPLE USAGE:**
+
+```js
+import 'braid-design-system/reset';
+
+import apac from 'braid-design-system/themes/apac';
+import { BraidProvider } from 'braid-design-system';
+
+import React from 'react';
+
+// This will wrap every story in a BraidProvider
+export const decorators = [
```

Comment:
Having two ways of declaring decorators in the same changelog entry might be confusing. I'd suggest merging the two changesets. The other one doesn't stand on its own.

Reply:
https://github.com/seek-oss/sku/pull/792/commits/2c92ac8447f0f69efeb05da0f7fb087bf6f9975b